pyproject.toml

```diff
@@ -27,15 +38,15 @@ requires = ["poetry-core"]
 build-backend = "poetry.core.masonry.api"
 
 [tool.pyright]
-reportMissingTypeStubs = false  # Changed from true
+reportMissingTypeStubs = false
 strict = ["nhs"]
 reportUnknownParameterType = true
 reportUntypedFunctionDecorator = true
 typeCheckingMode = "basic"
 pythonVersion = "3.12"
 include = ["**/*.py"]
 exclude = ["**/node_modules", "**/__pycache__"]
-ignore = ["syncdata.py", "src/oldstuff"]  # Changed to ignore the specific file with dirsync import
+ignore = ["syncdata.py", "src/oldstuff", "nhs/data/geography.py"]
 defineConstant = { DEBUG = true }
 strictListInference = true
```

Comment:
Also please don't ignore the file if it's giving you errors, that goes against the point of using type checkers in the first place. To avoid being flooded with type errors, develop with the type checking on so it'll give you errors when you write the code as opposed to all at once towards the end.